### PR TITLE
[New][Author] Add angular-patternfly w/ npm auto-update

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
# Profile of the lib
 * Git repository (required): https://github.com/patternfly/angular-patternfly
 * Official website (optional, not the repository): http://www.patternfly.org/angular-patternfly/
 * NPM package url (optional): https://www.npmjs.com/package/angular-patternfly
 * License and its reference: Apache License 2.0 https://github.com/patternfly/angular-patternfly/blob/master/LICENSE.txt
 * GitHub / Bitbucket popularity (required):
   - Count of watchers:26
   - Count of stars: 102
   - Count of forks: 82
 * NPM download stats (optional):
   - Downloads in the last day: 93
   - Downloads in the last week: 1167
   - Downloads in the last month: 5017

# Essential checklist
 * [ x] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
@jeff-phillips-18 thanks for your PR! @sashberd would you like to help review it!?

**Comment by jeff-phillips-18** [comment]
@PeterDaveHello I've removed the version. Please let me know if there are any further changes required. Thanks for your help.

**Comment by PeterDaveHello** [comment]
@jeff-phillips-18 thanks, will take a look soon!

**Comment by PeterDaveHello** [comment]
@jeff-phillips-18 69 releases on npm registry but 112 on GitHub, would that be a concern? We usually try to use a source with more versions(though once we added those the versions, there is no difference) and update more frequently, fast(if the different sources have that difference), I'm not sure if some tagged releases on GitHub may not be usable or something like that, as you are the author, I think it's easier to ask you directly, thanks.

**Comment by jeff-phillips-18** [comment]
@PeterDaveHello I believe that is just due to creating GitHub releases before we started publishing to NPM.

**Comment by PeterDaveHello** [comment]
@jeff-phillips-18 okay so for the recent versions, they should be the same, am I right?

**Comment by jeff-phillips-18** [comment]
@PeterDaveHello Yes.

**Comment by PeterDaveHello** [comment]
@jeff-phillips-18 cool, thanks! I'll manually add those missing versions from git repo.

**Comment by jeff-phillips-18** [comment]
@PeterDaveHello Any chance you can do the rebase? I'm afraid if I pull it all down to my fork and re-push I will run into the push issue I had before with http.postBuffer never being large enough to push.

**Comment by PeterDaveHello** [comment]
@jeff-phillips-18 of course I will do it, thanks for asking!

**Comment by PeterDaveHello** [comment]
@maruilian11 I think you should do it as you added something, any reason not to?

**Comment by jeff-phillips-18** [comment]
@PeterDaveHello Is there something I can do to help here?

**Comment by jeff-phillips-18** [comment]
@maruilian11 Should I attempt the rebase?

**Comment by PeterDaveHello** [comment]
@jeff-phillips-18 sorry for pending, we'll review this PR ASAP!

File: ajax/libs/angular-patternfly/package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "url": "git+https://github.com/patternfly/angular-patternfly.git"
+  },
+  "homepage": "https://www.patternfly.org/angular-patternfly",
+  "npmName": "angular-patternfly",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "!(docs|less)/**/*",
+        "angular-paternfly*"
```
Comment (sufuf3) [inline ajax/libs/angular-patternfly/package.json]: This pattern can't get `angular-patternfly.js` & `angular-patternfly.min.js`
Need to modify from `angular-paternfly*` to `angular-patternfly*` (add a `t` --> angular-pat**t**ernfly*)
Thank you.

**Comment by jeff-phillips-18** [comment]
Thank you @sufuf3

**Comment by jeff-phillips-18** [comment]
Anything I can do to help this along?

**Comment by sufuf3** [comment]
Let's wait for @extend1994 's review.
Thanks

**Comment by PeterDaveHello** [comment]
Thank you all for your contributions!

**Pull request merged**